src/js/gallery.js

```diff
@@ -15,6 +15,7 @@ const KEY = '79ea8908d5d0aaabd49d601dd35d503a';
 const LANG = 'en-US';
 export let page = 1;
 let queryValue = '';
+let whatToFetch = 'trending/movie/week?';
 
 const searchParams = () =>
   new URLSearchParams({
```

Comment:
W gruncie rzeczy to czy queryValue jest w ogóle potrzebne przy wyświetlaniu najpopularniejszych filmów?